recipes/cchardet/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cchardet" %}
+{% set version = "1.0.0" %}
+{% set md5 = "353f5ca2bc445cef5f103a9a7c2f7599" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/14/4f/7570c170110a79290824a683f92043ecc048f851e57a2b2e223a9fe5e8c2/cchardet-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 2
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for Python 2.7, 3.4, and above, for all major OSs.
+  # Add the line "skip: True  # [py<34]" (for example) to limit to Python 3.4 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+
+requirements:
+  build:
```

Comment:
Please add `toolchain`. This configures our compilers. Details at that feedstock.